src/estimators.jl

```diff
@@ -371,14 +366,14 @@ end
         max_cat_threshold = 32,
         cat_l2 = 10.0,
         cat_smooth = 10.0,
-        metric = [\"multi_logloss\"],
+        metric = [""],
         metric_freq = 1,
-        is_training_metric = false,
+        is_provide_training_metric = false,
         eval_at = Int[1, 2, 3, 4, 5],
         num_machines = 1,
         local_listen_port = 12400,
         time_out = 120,
-        machine_list_file = \"\",
+        machine_list_filename = \"\",
         num_class = 2,
```

Comment:
This does not match the default in the documentation but it' s because in the documentation also has the default objective set to regression for which num_class is also defaulted to 1.
In LighGBM.jl it's been decided to have now 3 estimators where the LGBMClassification is defaulted to multiclass (not binary) hence this default num_class.

Reply:
Just to clarify on this the regression task does not use num_class which is required for classification task. The default in this estimator differs from the default in the official docs because the classifier in LightGBM.jl is defaulted to multiclass objective.